src/styles/blocks/card.css

```diff
@@ -0,0 +1,37 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  /* box-sizing: border-box; */
+}
+
+.card__imac {
+  margin-bottom: 40px;
+  margin-top: 33px;
+}
+
+.card__title {
+  margin: 0 0 4px 17px;
+  align-self: flex-start;
+}
+
+.card__product-code {
+  display: block;
+  align-self: flex-start;
+  margin-left: 17px;
+  margin-top: 0;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  display: flex;
+  align-self: flex-start;
+  margin-top: 0;
+  margin-left: 17px;
+  margin-bottom: 16px;
```

Comment:
combine it with [`margin`](https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax) property. Please fix all cases if present

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/210100088-03c1d853-2115-4996-b9a0-67630c0f53e9.png)